automation/README.md

```diff
@@ -0,0 +1,83 @@
+# Automation
+
+The Gadget's automation service is a custom container that allows the Gadget to communicate with the outside world. It interprets decisions from the model pipeline and sends the needed messages out. Because each inspection application is different and may need to communicate with anything using any protocol the implementation of the automation class is custom. The automation server is the base which the service is build on. The automation server takes as inputs two files, an automation class python file and an automation configs json file. It takes both those files, adds the configs to the database and uses the custom code in the automation class to send messages to an automation machine. The automation server is given these files through env variables **AUTOMATION_PATH** (the absolute path to the folder inside the container that holds the files), **AUTOMATION_CLASS** (the file and class name the automation server uses to import the automation class), and **AUTOMATION_DEFINITION_JSON** (the name of the JSON file).
```

Comment:
``Because each inspection application is different and may need to communicate with anything using any protocol the implementation of the automation class is custom``
sounds like two sentences. might be better to split into two instead of single sentence.